src/XIVLauncher.Core/Components/SettingsPage/Tabs/SettingsTabSteamTool.cs

```diff
@@ -0,0 +1,140 @@
+﻿using System.Numerics;
+using System.Runtime.InteropServices;
+using System.IO;
+using ImGuiNET;
+using XIVLauncher.Common.Unix.Compatibility;
+using XIVLauncher.Common.Util;
+using XIVLauncher.Core.UnixCompatibility;
+
+namespace XIVLauncher.Core.Components.SettingsPage.Tabs;
+
+public class SettingsTabSteamTool : SettingsTab
+{
+    private SettingsEntry<string> steamPath;
+    
+    private SettingsEntry<string> steamFlatpakPath;
+
+    private bool steamInstalled = SteamCompatibilityTool.IsSteamInstalled;
+
+    private bool steamToolInstalled = SteamCompatibilityTool.IsSteamToolInstalled;
+    
+    private bool steamFlatpakInstalled = SteamCompatibilityTool.IsSteamFlatpakInstalled;
+    
+    private bool steamFlatpakToolInstalled = SteamCompatibilityTool.IsSteamFlatpakToolInstalled;
+
+    public SettingsTabSteamTool()
+    {
+        Entries = new SettingsEntry[]
+        {
+            steamPath = new SettingsEntry<string>("Steam Path (native install)", "Path to the native steam config files. Only change this if you have your steam config stored somewhere else.",
+                () => Program.Config.SteamPath ?? Path.Combine(CoreEnvironmentSettings.HOME, ".local", "share", "Steam"), s => Program.Config.SteamPath = s),
+            steamFlatpakPath = new SettingsEntry<string>("Steam Path (flatpak install)", "Path to the flatpak steam config files. Only change this if you have your steam config stored somewhere else.",
+                () => Program.Config.SteamFlatpakPath ?? Path.Combine(CoreEnvironmentSettings.HOME, ".var", "app", "com.valvesoftware.Steam", "data", "Steam" ), s => Program.Config.SteamFlatpakPath = s)
+            {    
+                CheckVisibility = () => CoreEnvironmentSettings.IsDeck != true && SteamCompatibilityTool.IsSteamFlatpakInstalled,
+            },
+        };
+    }
+
+    public override SettingsEntry[] Entries { get; }
+
+    public override bool IsUnixExclusive => true;
+
+    public override string Title => "Steam Tool";
+
+    public override void Draw()
+    {
+        if (CoreEnvironmentSettings.IsSteamCompatTool)
+        {
+            ImGui.Dummy(new Vector2(10));
+            ImGui.Text("You are currently running XIVLauncher.Core as a Steam compatibility tool.");
+            ImGui.Dummy(new Vector2(10));
+            ImGui.Text("If you are trying to upgrade, you must first update your local install of XIVLauncher.Core. Then launch the local" +
+                        "\nversion, navigate back to this tab, and re-install as a Steam compatibility tool.");
+            ImGui.Text("\nIf you are trying to uninstall, you should likewise launch the native version of XIVLauncher, and click the appropriate" +
+                        "\nuninstall button.");
+            return;
+        }
+        ImGui.Text("\nUse this tab to install XIVLauncher.Core as a Steam compatibility tool.");
```

Comment:
I think it would be a good idea to have the opening explain quickly what a "steam tool" is and why a user might want to use it.